Integrations/GoogleDocs/GoogleDocs.py

```diff
@@ -0,0 +1,395 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import requests
+import httplib2
+import urllib.parse
+from oauth2client import service_account
+from googleapiclient import discovery
+
+import typing
+from collections import defaultdict
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+SCOPES = ['https://www.googleapis.com/auth/documents']
+
+ACTION_TO_FUNCTION = {
+    'createNamedRange': 'create_named_range',
+    'createParagraphBullets': 'create_paragraph_bullets',
+    'deleteContentRange': 'delete_content_range',
+    'deleteNamedRange': 'delete_named_range',
+    'deleteParagraphBullets': 'delete_paragraph_bullets',
+    'deletePositionedObject': 'delete_position_object',
+    'deleteTableColumn': 'delete_table_column',
+    'deleteTableRow': 'delete_table_row',
+    'insertInlineImage': 'insert_inline_image',
+    'insertPageBreak': 'insert_page_break',
+    'insertTable': 'insert_table',
+    'insertTableColumn': 'insert_table_column',
+    'insertTableRow': 'insert_Table_row',
+    'insertText': 'insert_text',
+    'replaceAllText': 'replace_all_text'
+}
+''' HELPER FUNCTIONS '''
+
+
+def parse_actions(actions: str):
+    """Destructs action1{param1,param2,...};action2{param1,param2,...}... to a dictionary where keys are action type and
+      values are function params"""
+    parsed_actions = dict()
+    actions = actions.split(';')
+    for action in actions:
+        action_type, params = action.split('{')
+        params = params[:-1]
+        params = params.split(',')
+        parsed_actions[action_type] = params
+    return parsed_actions
+
+
+def log_error(f):
```

Comment:
@SgtJahnoon very nice.
@Itay4 do we have such decorator already? If not perhaps we can add it to common server python